src/ohri-form.component.tsx

```diff
@@ -239,90 +258,104 @@ const OHRIForm: React.FC<OHRIFormProps> = ({
         handleFormSubmit(values);
         setSubmitting(false);
       }}>
-      {props => (
-        <Form className={`cds--form no-padding ${styles.ohriForm}`} ref={ref}>
-          {isLoadingPatient || isLoadingFormJson ? (
-            <LoadingIcon />
-          ) : (
-            <div className={styles.ohriFormContainer}>
-              {isLoadingFormDependencies && (
-                <div className={styles.loader}>
-                  <LinearLoader />
-                </div>
-              )}
-              <div className={styles.ohriFormBody}>
-                {showSidebar && (
-                  <OHRIFormSidebar
-                    isFormSubmitting={isSubmitting}
-                    pagesWithErrors={pagesWithErrors}
-                    scrollablePages={scrollablePages}
-                    selectedPage={selectedPage}
-                    mode={mode}
-                    onCancel={onCancel}
-                    handleClose={handleClose}
-                    values={props.values}
-                    setValues={props.setValues}
-                    allowUnspecifiedAll={formJson.allowUnspecifiedAll}
-                    defaultPage={formJson.defaultPage}
-                  />
+      {props => {
+        setIsFormTouched(props.dirty);
```

Comment:
Sets `isFormTouched` to the value of `props.dirty`, which gets determined by whether the form field has been touched or not.